artifacts/CAMARA_common.yaml

```diff
@@ -265,6 +265,16 @@ components:
       status: 501
       code: NOT_IMPLEMENTED
       message: This functionality is not implemented yet
+  NotSupported:
+   description: Not Supported
+   content:
+    application/json:
+     schema:
+      $ref: "#/components/schemas/ErrorInfo"
+     example:
+      status: 501
+      code: NOT_SUPPORTED

```

Comment:
```suggestion
      code: PHONE_NUMBER_NOT_SUPPORTED

```

Reply:
@PedroDiez
Suggest not to be specific to "Phone_Number", so this error can be reused.